src/services/project.ts

```diff
@@ -27,10 +27,7 @@ async function getProjectColumns(projectId: string): Promise<Project> {
       id: projectIdQuery,
     },
     relations: {
-      columns: {
-        id: true,
-        sort: true,
-      },
+      columns: true,
```

Comment:
Here, I have no idea why typeorm is doing such a poor job. It was recommended syntax and lint and ts were happy with what we had but it wasn't working in the test suite. It looks like your previous implementation was the "suggested" way but I switched to just boolean. I'll take all key/values from `Column` but I think it's trivial since we won't have thousands of items queried.